### PR TITLE
Template for the issue tracker

### PR DESCRIPTION
**Pull request by nanjekyejoannah**
This is a proposed template covering details for a bug report, as discussed in our last call with @lpereira95 but also noted in #1597

### CODE REVIEW

**Review by lpereira95 — COMMENTED** [review]
@nanjekyejoannah can you make it slightly closer to sklearn's ([here](https://github.com/scikit-learn/scikit-learn/issues/new?assignees=&labels=Bug%2CNeeds+Triage&template=bug_report.yml) and [configuration](https://github.com/scikit-learn/scikit-learn/blob/main/.github/ISSUE_TEMPLATE/bug_report.yml))?

I like specially the fact that they have a code block where to write the code to reproduce, and the fields "Expected results" and "Actual results" (I may prefer the word "behavior" that you use). The "Describe the bug" is also very meaningful.

**Comment by nanjekyejoannah** [comment]
I have done some modifications, PTAL

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -4,38 +4,68 @@ title: "Bug: "
...
+- type: markdown
+  attributes:
+    value: >
+      
+        Thank you for opening this issue!
 
+        If you're new to Geomstats, consider the following options first:
+
+        - reading the Geomstats documentation: https://geomstats.github.io/
+        - searching our existing issue tracker for a similar issue (https://github.com/geomstats/geomstats) to see if
```
Comment (lpereira95) [inline .github/ISSUE_TEMPLATE/bug-report.yml:16]: https://github.com/geomstats/geomstats/issues instead?

**Review by lpereira95 — APPROVED** [review]
Thanks @nanjekyejoannah, it looks great!

I think we should update issues link and merge.

**Comment by nanjekyejoannah** [comment]
I have updated, thank, I can now use it in the docs, i guess.

**Comment by lpereira95** [comment]
Thanks @nanjekyejoannah!
